test/parallel/test-https-localaddress.js

```diff
@@ -39,7 +39,7 @@ const options = {
 
 const server = https.createServer(options, function(req, res) {
   console.log(`Connect from: ${req.connection.remoteAddress}`);
-  assert.strictEqual(req.connection.remoteAddress, '127.0.0.2');
+  assert.match(req.connection.remoteAddress, /^127\.0\.0\.1$|^::1$/);
```

Comment:
Should `127.0.0.2` be expected instead of `127.0.0.1`?
```suggestion
  assert.match(req.connection.remoteAddress, /^(127\.0\.0\.2|::1)$/);
```

Reply:
```suggestion
  assert.match(req.connection.remoteAddress, /^(127\.0\.0\.1|127\.0\.0\.2|::1)$/);
```